extensions/pom.xml

```diff
@@ -95,6 +95,8 @@
 
         <!-- Additional languages -->
         <module>kotlin</module>
+
```

Comment:
Perhaps we can add a comment which indicates what's the extension is for. Something like - `<!-- Database migrations -->`

Reply:
Cool, I will do it